### DISCUSSION TITLE
How to use text attribute or mtext attribute to bold font in ezdxf？

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#711] [Q&A]
How to use text attribute or mtext attribute to bold font in ezdxf？

best wishes

**Comment by mozman** [comment]
Tutorial  for `MText` and the helper tool `MTextEditor`: https://ezdxf.mozman.at/docs/tutorials/mtext.html#tut-mtext

**Reply by luoyu-123** [reply]
Dear Mr. Mozman:
    Thank you very much for the reply, I checked the documentation and didn't find any relevant attributes for bolding the same font. The image below is the effect I want

![image](https://user-images.githubusercontent.com/59160360/181447805-2ab39d79-ded9-4118-b537-b5a60ffaec56.png)

**Comment by mozman** [comment]
This feature is only supported by the  `MTEXT` entity and only by setting the font properties by inline codes like in this content string: 

`"abc {\fArial|b1|i0|c0|p0;abc} abc"`,  `b1` switches bold on, `b0` switches bold off, same for italic by `i1` and `i0`.

![image](https://user-images.githubusercontent.com/4230966/181466249-aa3e92a5-975c-4bb2-9ee8-cdcb626387b6.png)

**Reply by luoyu-123** [reply]
thank you very much! Dear Mr. Mozman.
Where can I find more inline code for mtext entity operations in ezdxf

**Comment by mozman** [comment]
Table of known inline-codes: https://ezdxf.mozman.at/docs/dxfentities/mtext.html#mtext-inline-codes

Helper class `MTextEditor`: https://ezdxf.mozman.at/docs/tools/text.html#ezdxf.tools.text.MTextEditor

Tutorial: https://ezdxf.mozman.at/docs/tutorials/mtext.html#mtexteditor
